### PR TITLE
faq: minor fix

### PR DESCRIPTION
**Pull request by overvenus**
PTAL @andelf @siddontang @huachaohuang

### CODE REVIEW

File: faq.md [hunk 1]
```diff
@@ -122,4 +122,4 @@ PD 的大部分 API 需要在初始化 TiKV 集群以后才能使用，如果在
 
 ## PD 启动报错：Etcd cluster ID mismatch
 
-PD 启动参数中的 \-initial\-cluster 包含了某个不属于该集群的成员。遇到这个错误时请检查各个成员的所属集群，剔除错误的成员后即可正常启动。
+PD 启动参数中的 \-\-initial\-cluster 包含了某个不属于该集群的成员。遇到这个错误时请检查各个成员的所属集群，剔除错误的成员后即可正常启动。
```
Comment (siddontang) [inline faq.md]:
```
`--initial-cluster`
```

Comment (overvenus) [inline faq.md]: Previous style is `\-\-`.

```md
## TiDB 的 lease 参数应该如何设置？

启动 TiDB Server 时，需要通过命令行参数设置 lease 参数（\-\-lease=60），
```

**Comment by andelf** [comment]
LGTM

**Comment by andelf** [comment]
reset LGTM.

File: faq.md [hunk 2]
```diff
@@ -110,7 +110,7 @@ TiDB 支持绝大多数 MySQL 语法，一般不需要修改代码。我们提
 
 ## 为什么 TiKV 数据目录不见了
 
-TiKV 的 \-\-store 参数默认值为 `/tmp/tikv/store`，在某些虚拟机中，重启操作系统会删除 /tmp 目录下的数据，推荐通过 --store 参数显式设置 TiKV 数据目录。
+TiKV 的 `--store` 参数默认值为 `/tmp/tikv/store`，在某些虚拟机中，重启操作系统会删除 /tmp 目录下的数据，推荐通过 --store 参数显式设置 TiKV 数据目录。
```
Comment (andelf) [inline faq.md]: `--store`

File: faq.md [hunk 3]
```diff
@@ -122,4 +122,4 @@ PD 的大部分 API 需要在初始化 TiKV 集群以后才能使用，如果在
 
 ## PD 启动报错：Etcd cluster ID mismatch
```
Comment (andelf) [inline faq.md]: `s/Etcd/etcd/`

What's the original error?

**Comment by siddontang** [comment]
Ping @overvenus

**Comment by andelf** [comment]
LGTM

**Pull request merged**